packages/nextjs-mf/src/internal.ts

```diff
@@ -93,40 +93,36 @@ export const DEFAULT_SHARE_SCOPE: SharedObject = {
 };
```

Comment:
The function 'getDelegates' can be simplified by using the filter method instead of reduce. This can improve readability and maintainability.